src/api/common/utils.js

```diff
@@ -17,7 +17,7 @@ function xrpToDrops(xrp: string): string {
   return (new BigNumber(xrp)).times(1000000.0).floor().toString();
 }
 
-function toRippledAmount(amount: Amount): string|Amount {
+function toRippledAmount(amount: AmountRippled): string|AmountRippled {
```

Comment:
This doesn't seem right. It's taking an `AmountRippled` as input ? I believe this function takes a RippleAPI amount and converts it to the json form as spoken by rippled.